app/models/ems_cluster.rb

```diff
@@ -119,15 +119,6 @@ def total_vcpus
   alias_method :all_miq_templates,      :miq_templates
   alias_method :all_miq_template_ids,   :miq_template_ids
 
-  # Host relationship methods
-  def failover_hosts(_options = {})
-    hosts.select(&:failover)
-  end
-
-  def failover_host_ids
```

Comment:
do we get this method from the has_many?  I forget if it singularlizes it for you